docs/tutorial/dingpu.md

```diff
@@ -2,6 +2,8 @@
 
 ## Basic chat usage
```

Comment:
Why don't you have to add a command line flag to indicate to use that node as store/filter node? Doesn't make sense to me